TimeCalibration/TimeCalibration.h

```diff
@@ -0,0 +1,64 @@
+ /**
+ *  @copyright Copyright 2016 The J-PET Framework Authors. All rights reserved.
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may find a copy of the License in the LICENCE file.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Purpose: Time calibration with data measured with reference detector
+ *  @file TimeCalibration.h
+ */
+
+#ifndef TimeCalibration_H 
+#define TimeCalibration_H 
+#include <JPetTask/JPetTask.h>
+#include <JPetHit/JPetHit.h>
+#include <JPetRawSignal/JPetRawSignal.h>
+#include "LargeBarrelMapping.h"
```

Comment:
Please do not use LargeBarrelMapping and use JPetGeomMapping from the framework instead.
The interface is almost the same:
http://sphinx.if.uj.edu.pl/framework/doc/classJPetGeomMapping.html

You can find an example of JPetGeomMapping usage here:
https://github.com/alekgajos/j-pet-framework-examples/blob/timeWindow/LargeBarrelAnalysisExtended/TimeCalibLoader.cpp

Please also remove the files of LargeBarrelMapping from this pull request.

Reply:
Done